convert2rhel/actions/system_checks/check_firewalld_availability.py

```diff
@@ -0,0 +1,133 @@
+# Copyright(C) 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+import logging
+import os
+import re
+
+from convert2rhel import actions, systeminfo
+from convert2rhel.systeminfo import system_info
+
+
+logger = logging.getLogger(__name__)
+
+# Path to the firewalld config file.
+FIREWALLD_CONFIG_FILE = "/etc/firewalld/firewalld.conf"
+
+# Regex that identify if CleanupModulesOnExit is set to either yes or true
+# considering blank spaces in between the value and case insensitive.
+CLEANUP_MODULES_ON_EXIT_REGEX = re.compile(r"(?i)CleanupModulesOnExit\s*=\s*(yes|true)")
+
+
+def _is_modules_cleanup_enabled():
+    """Verify firewalld modules cleanup config is enabled.
+
+    :returns: Whether or not the CleanupModulesOnExit is set to true in
+        firewalld config.
+    :rtype: bool
+    """
+    # Return false if the config file does not exist. Either it means that
+    # firewalld is not installed or the config file was removed.
+    if not os.path.exists(FIREWALLD_CONFIG_FILE):
+        logger.debug("%s does not exist." % FIREWALLD_CONFIG_FILE)
+        return False
```

Comment:
```suggestion
        return True
```
This needs to be True because when the config file does not exist, firewalld will use a default configuration (that means CleanupModulesOnExit=yes).
```
WARNING: [Errno 2] No such file or directory: '/etc/firewalld/firewalld.conf'
firewalld[8585]: WARNING: Using fallback firewalld configuration settings.
```